fixtures/src/components/controlled/checkbox-group.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import { Checkbox, CheckboxGroup } from 'carbon-react/lib/__experimental__/components/checkbox';
+import { LogConsumer } from '../log';
+
+const ControlledCheckbox = () => {
+  const [checked, setChecked] = useState({
+    controlled_checkbox_group_default_two: true
+  });
+  return (
+    <LogConsumer>
+      {(log) => {
+        const onChange = (e) => {
+          log(e, { method: 'onChange' });
+          setChecked({
+            ...checked,
+            [e.target.name]: e.target.checked
+          });
+        };
+        const onBlur = e => log(e, { method: 'onBlur' });
+        return (
+          <React.Fragment>
+            <h1>Controlled Checkbox Group</h1>
+            <h2>Unchecked</h2>
+            <div id='controlled_checkbox_group'>
+              <ul>
+                <li>onChange handler should update the log when each checkbox is checked</li>
```

Comment:
Presumably the `onChange` handler should also update the log when each checkbox gets unchecked too? So maybe something like this would be clearer:
```suggestion
                <li>onChange handler should update the log when each checkbox is changed/toggled</li>
```